tasks/eva_2779/tests/test_correct_assembly_error_in_rat.py

```diff
@@ -0,0 +1,120 @@
+                        "contig": "CM026996.1",
+                        "start": 39058078,
+                        "type": "SNV",
+                        "hashedMessage": "AF611B8ED3BE813B427AD7E48D266D3DB5F013B8",
+                        "accession": 198834536,
+                        "version": 1,
+                },
+                {
+                    "asm": "GCA_015227675.1",
+                    "tax": 10116,
+                    "contig": "CM026996.1",
+                    "start": 39058077,
+                    "type": "SNV",
+                    "hashedMessage": "AF611B8ED3BE813B427AD7E48D266D3DB5F013B8",
+                    "accession": 198834532,
+                    "version": 1,
+                }
+            ]
+        }
+
+        self.connection_handle[self.accession_db][self.submitted_variants_collection].drop()
+        self.connection_handle[self.accession_db][self.clustered_variants_collection].drop()
+        self.connection_handle[self.accession_db][self.clustered_variants_operation_collection].drop()
+        self.connection_handle[self.accession_db][self.submitted_variants_collection].insert_many([ss1])
+        self.connection_handle[self.accession_db][self.clustered_variants_collection].insert_many([rs1])
+        self.connection_handle[self.accession_db][self.clustered_variants_operation_collection].insert_many([cvoe1])
+
+    def tearDown(self) -> None:
+        self.connection_handle[self.accession_db][self.submitted_variants_collection].drop()
+        self.connection_handle[self.accession_db][self.clustered_variants_collection].drop()
+        self.connection_handle.close()
+
+    def test_replace_document_with_correct_information(self):
+        replace_document_with_correct_information(
+            self.mongo_db, "submittedVariantEntity", get_submitted_SHA1,
+            {'seq': 'GCA_015227675.1', 'contig': 'CM026996.1'},
+            {'seq': 'GCA_015227675.2', 'contig': 'AY172581.1'},
+            5
+        )
+        cursor = self.connection_handle[self.accession_db][self.submitted_variants_collection].find()
+        document = cursor.__next__()
+        assert document['seq'] == 'GCA_015227675.2'
+        assert document['contig'] == 'AY172581.1'
+
+    def test_update_operation_entities(self):
+        update_operation_entities(
+            self.mongo_db, 'clusteredVariantOperationEntity', get_clustered_SHA1,
+            {'inactiveObjects.asm': 'GCA_015227675.1', 'eventType': 'MERGED'},
+            {'inactiveObjects.asm': 'GCA_015227675.2',
+             'reason': lambda x: x.replace('GCA_015227675.1', 'GCA_015227675.2')},
+            5
+        )
+        cursor = self.connection_handle[self.accession_db][self.clustered_variants_operation_collection].find()
+        document = cursor.__next__()
+        assert document['reason'] == 'After remapping to GCA_015227675.2, RS IDs mapped to the same locus.'
+        assert document['inactiveObjects'][0]['asm'] == 'GCA_015227675.2'
```

Comment:
Might want to also check the inactive object's `hashedMessage` is set correctly.